server/src/main/java/fr/uga/l3miage/tp4/models/UserEntity.java

```diff
@@ -0,0 +1,21 @@
+package fr.uga.l3miage.tp4.models;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Setter
+@Getter
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
+public abstract class UserEntity {
```

Comment:
il vous manque le `@MappedSuperclass` et ne pas déclarer le `@Entity` ici puisque vous le faite dans les classes filles, ou sinon vous faite l'inverse vous le déclarer ici, et pas en dessous
```suggestion
@Setter
@Getter
@MappedSuperclass
@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
public abstract class UserEntity {
```